### DISCUSSION TITLE
How to use selected entity in Template card

### DISCUSSION THREAD

**Discussion by j-loewen** [piitaya/lovelace-mushroom#236] [General]
Hello,

in Release [v1.2.4](https://github.com/piitaya/lovelace-mushroom/releases/tag/v1.2.4) a selector for the entity was added to the Template card. My Question is: How can I use the selected entity in the templates of the Template card. In most cases I do know the entity but if I use the Template card with the "custom:auto-entities" card I don't have this information.

Thanks in advance

**Comment by piitaya** [comment]
You can use the entity in template like this :
```
{{ states(entity) }}
```
I agree, the doc needs to be updated for template  😅
